packages/local_auth/local_auth_ios/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -216,26 +216,28 @@ - (void)handleAuthReplyWithSuccess:(BOOL)success
     result(@YES);
   } else {
     switch (error.code) {
-      case LAErrorPasscodeNotSet:
+      case LAErrorSystemCancel:
```

Comment:
You could put this back to where it was at the bottom to avoid dirtying the git history and just add the `result(@NO)` part.